js-audit-allow-list.json

```diff
@@ -0,0 +1,129 @@
+{
+  "trim-newlines": [
+    {
+      "title": "Regex DOS",
+      "cve": "CVE-2021-33623",
+      "reason": "Regex DOS attacks aren't relevant in our devDependencies",
+      "reviewer": "christianklammer",
+      "whenSubDependencyOf": ["lerna", "node-sass"]
+    }
+  ],
+  "normalize-url": [
+    {
+      "title": "Regex DOS",
+      "cve": "CVE-2021-33502",
+      "reason": "Dev dependency",
+      "reviewer": "christianklammer",
+      "whenSubDependencyOf": ["gatsby", "gatsby-source-filesystem"]
+    }
+  ],
+  "url-parse": [
+    {
+      "title": "URL Redirection to Untrusted Site",
+      "cve": "CVE-2021-3664",
+      "reason": "Dev dependency",
```

Comment:
Do you know if this dependency is used in the generated site or not?

Given the nature of Kaizen (static site, separate domain, no user data) this feels sufficiently low risk to me but it's not clear that Gatsby is always a "devDependency"

Reply:
Yes, that's definitely a problem. My "hope" is, as we're using Gatsby as a static site generator, little to none of its own code is actually bundled in with the generated code.
The `url-parse` package is part of a websocket package which I think is just used for local dev. Downloaded the Gatsby repo and also had a look at the build artifact.